llvm/include/llvm/Analysis/CmpInstAnalysis.h

```diff
@@ -92,18 +92,21 @@ namespace llvm {
   Constant *getPredForFCmpCode(unsigned Code, Type *OpTy,
                                CmpInst::Predicate &Pred);
 
-  /// Represents the operation icmp (X & Mask) pred 0, where pred can only be
+  /// Represents the operation icmp (X & Mask) pred Cmp, where pred can only be
```

Comment:
Just an initial thought, but imo `Cmp` implies we are doing `(icmp pred (X & Mask), (another icmp)`. How about `Val`?

Reply:
Sure :)